refactor: [BD-46] Component generator refactoring (#3076)
openedx/paragon: README.md

##### 4. (Optional) Add styles to your component.

If your component requires additional styling (which most likely is the case), edit created SCSS style sheet in your
component's directory `/src/MyComponent/index.scss` which by default contains an empty class for your component.

> If you wish to use CSS variables (which is the preferred way of styling the components since values can be
> easily overridden and customized by the consumers of Paragon), create specific design tokens in `tokens` directory (in the `core` and `themes` directories) and then generate (`npm run build-tokens`) CSS variables for your component.

Reviewer: I think it'd be nice to keep the design token documentation in the `tokens` directory and link to it instead. 

```suggestion
If you wish to use CSS variables (which is the preferred way of styling the components since values can be
easily overridden and customized by the consumers of Paragon), you can do so by utilizing [design tokens](/tokens).
```

this way we can keep the documentation in one place and we don't need to remember all the places it needs to be updated if it is changed.

Author: I edited it, thanks! Your proposal looks more concise 💯
